README.md

```diff
@@ -8,6 +8,41 @@ Yet another note taking tool.
 cargo tauri dev
 ```
 
+## Setup environment
+This guide focuses on how to setup your development system based on ubuntu:jammy.
+
+1. Install dependencies
+    ```bash
+    sudo apt-get update
+    sudo apt-get install -y \
+        libwebkit2gtk-4.0-dev \
+        build-essential \
+        curl \
+        wget \
+        file \
```

Comment:
We do not depend on these packages (see [build workflow](https://github.com/falk-werner/note.rs/blob/main/.github/workflows/build.yaml)), please remove it.
```suggestion
```

Reply:
This packages are removed with commit c337707716c614b8271430b3999e47c3e33cc656.